pkgs/development/python-modules/jinja2-ansible-filters/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, buildPythonPackage, fetchPypi
+, jinja2
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "jinja2-ansible-filters";
+  version = "1.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-ek5vhv+oFIQectDDwix+cnvLg4fXuF5CJKEYFjSjYbQ=";
+  };
+
+  # VERSION is not included in sdist -_-
+  postPatch = ''
+    echo ${version} > VERSION
+  '';
+
+  propagatedBuildInputs = [
+    jinja2
+    pyyaml
+  ];
+
+  # no tests include in sdist, and source not available
+  doCheck = false;
```

Comment:
The upstream repo contains a couple of [`tests`](https://gitlab.com/dreamer-labs/libraries/jinja2-ansible-filters/-/tree/master/tests).